common/lib/transport/connectionmanager.js

```diff
@@ -101,6 +101,7 @@ var ConnectionManager = (function() {
 		}
 		params.v = Defaults.apiVersion;
 		params.lib = Defaults.libstring;
+		params.agent = Defaults.agent;
```

Comment:
Per RSC7b this is supposed to replace the lib param & header, surely we shouldn't send both at the same time?